CoreScriptsRoot/CoreScripts/NotificationScript2.lua

```diff
@@ -175,11 +182,11 @@ PopupText.Text = "This is a popup"
 PopupText.Parent = PopupFrame
 
 --[[ Helper Functions ]]--
-local insertNotifcation = nil
+local insertNotification = nil
```

Comment:
I don't remember why this was done, but looking at the code.... both functions call each other. insertNotification calls removeNotification in a delay() and removeNotification calls insertNotification if there are notifications in the overflow queue.

If we want to change this, then we'll maybe have to rewrite insert to possible fire an event when it needs to remove, and then have a listeners call remove.